eng/Version.Details.xml

```diff
@@ -12,27 +12,27 @@
       <Uri>https://github.com/xamarin/xamarin-android</Uri>
       <Sha>dc07f98a55b1597c834bf7b5ebd6b10223244583</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.MacCatalyst.Sdk" Version="16.4.7058">
+    <Dependency Name="Microsoft.MacCatalyst.Sdk" Version="16.4.7060">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>b1d56007abb11ba1c42f6da183e88591e4edb52d</Sha>
+      <Sha>edd059428ce051ae1a7d4092978e0c3ec6c7697d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.macOS.Sdk" Version="13.3.7058">
+    <Dependency Name="Microsoft.macOS.Sdk" Version="13.3.7060">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>b1d56007abb11ba1c42f6da183e88591e4edb52d</Sha>
+      <Sha>edd059428ce051ae1a7d4092978e0c3ec6c7697d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.iOS.Sdk" Version="16.4.7058">
+    <Dependency Name="Microsoft.iOS.Sdk" Version="16.4.7060">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>b1d56007abb11ba1c42f6da183e88591e4edb52d</Sha>
+      <Sha>edd059428ce051ae1a7d4092978e0c3ec6c7697d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.tvOS.Sdk" Version="16.4.7058">
+    <Dependency Name="Microsoft.tvOS.Sdk" Version="16.4.7060">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>b1d56007abb11ba1c42f6da183e88591e4edb52d</Sha>
+      <Sha>edd059428ce051ae1a7d4092978e0c3ec6c7697d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100" Version="7.0.5" CoherentParentDependency="Microsoft.NETCore.App.Ref">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100" Version="7.0.4" CoherentParentDependency="Microsoft.NETCore.App.Ref">
```

Comment:
We're at 7.0.5 here:

https://github.com/xamarin/xamarin-macios/blob/edd059428ce051ae1a7d4092978e0c3ec6c7697d/eng/Version.Details.xml#L21

but Android isn't:

https://github.com/xamarin/xamarin-android/blob/dc07f98a55b1597c834bf7b5ebd6b10223244583/eng/Version.Details.xml#L15

and since the dependency tree (CoherentParentDependency) goes like this:

* Microsoft.NETCore.App.Ref
* Microsoft.Android.Sdk.Windows

I'm guessing it's Android forcing this downgrade.

Which of course begs the question: how did it get to 7.0.5 in the first place?

Reply:
> Which of course begs the question: how did it get to 7.0.5 in the first place?

That happened here:

https://github.com/dotnet/maui/pull/14334

which was done manually, so that might have been the reason.

In any case, I don't believe this downgrade will affect anything (either accepted or not).

Reply:
If it's just emscripten, we can just ignore it because we don't use it (it just is a dependency of Mono). As long as workloads install & projects build, it's fine.

We are on 7.0.6 now here: https://github.com/xamarin/xamarin-android/blob/release/7.0.2xx/eng/Version.Details.xml